lab2

Rugelgen opened the pull request.

Rugelgen committed:
Add files via upload

TimurIslamgulov commented on ЛР2/2.py:
```diff
@@ -0,0 +1,67 @@
+# TODO написать класс Library
+class Library:
+
+    def __init__(self, books: list = [], id_spis: dict = {}):
```
а тут не нужен второй аргумент передавать вообще

TimurIslamgulov replied:
Написать класс Library, конструктор которого будет инициализировать следующие атрибуты:

books - Список книг
Конструктор должен принимать необязательный аргумент со значением по умолчанию. Если пользователь его не передал, то библиотека инициализируется с пустым списком книг.

Rugelgen replied:
Изменённый файл с верным кодом находится в том же pull request под названием "2.1.py"

TimurIslamgulov commented on ЛР2/2.py:
```diff
@@ -0,0 +1,67 @@
+            ident = last_book.id+1
+        return ident
+    def get_index_by_book_id(self, id):
+        if id in self.id_spis:
```
то есть это тоже неверно

Rugelgen committed:
Create 2.1

исправление атрибутов класса Library
id.spis убран из атрибутов Library, вследствие изменён весь код с его содержанием(без изменения функционала кода).

Rugelgen committed:
Create 2.1

исправление атрибутов класса Library
id.spis убран из атрибутов Library, вследствие изменён весь код с его содержанием(без изменения функционала кода).

Rugelgen committed:
Update 2.1

Rugelgen committed:
Update 2.1

исправление атрибутов класса Library
id.spis убран из атрибутов Library, вследствие изменён весь код с его содержанием(без изменения функционала кода).

Rugelgen committed:
Delete 2.1

Rugelgen committed:
Rename 2.1 to 2.1.py
